TheRocket/TheRocket.ipynb

```diff
@@ -26,59 +229,59 @@
     "discrete_segments = 100\n",
```

Comment:
Any reason to leave these general solver parameters as coded values instead of including them in the excel file as well?

Reply:
I figured that these are parameters to be tweaked during runtime of the simulations rather than variables for the different groups to input so it might be easier down the road for people working on the flight simulations to tweak these two directly in the notebook

Reply:
That's fair - thanks for explaining